### ISSUE TITLE
A readOnly mode

### ISSUE THREAD

**Issue by ritz078** [iqnivek/react-piano#27]
In this case no listeners like press etc. should work but it can still be controlled from outside by passing `playbackNotes `

**Comment by iqnivek** [comment]
Yeah, I think this makes sense to do. Perhaps instead of calling the prop `readOnly` we might want to call it something like `disableInput` or something, to map more closely to what it does. Alternatively, we might want to rename `props.disabled` to `props.loading` and then use `props.disabled` for disabling input (downside is that that would be a breaking change).

**Comment by kevinsqi** [comment]
@ritz078 I believe this is addressed in #36 with Piano.activeNotes, which I released in v3.0.0 today, which when set essentially makes the component readonly. Let me know if this doesn't meet the requirement here and I'll reopen this.

**Comment by agualdron** [comment]
CSS can be used to disable pointer events on a parent element

```css
.avoid-clicks {
  pointer-events: none;
}
```

with react styles, should look like this:
```js
<div style={{pointerEvents: "none"}}>
          <Piano
            noteRange={noteRange}
            width={300}
            disabled={false}
            playNote={playNote}
            stopNote={stopNote}
            activeNotes={activeNotes}
          />
</div>
```
